pkg/pgmodel/ingestor/samples-parser/parser.go

```diff
@@ -2,30 +2,35 @@
 // Please see the included NOTICE for copyright information and
 // LICENSE for a copy of the license.
 
-package ingestor
+package samples_parser
```

Comment:
I don't agree wiil this change. Also note this PR will have to go in after #571 so this has to be re-thought

Reply:
Yep, even i didnt like this, but that I guess was done to avoid cyclic import. Will rebase and see if this can be done in a better manner.

Reply:
Luckily, rebasing made this not a requirement. Hence, got rid of samples-parser package.